app/javascript/packs/bulk.js

```diff
@@ -293,10 +293,6 @@ Blacklight.onLoad(()=>{
 	$('#apply-apo-defaults-button').on('click', () => $('#apply_apo_defaults').show(400))
 	$('#add-workflow-button').on('click', () => $('#add_workflow').show(400))
 	$('#close-versions-button').on('click', () => $('#close').show(400))
-  $('#show_tags').on('click', () => {
```

Comment:
It looks like there may still be a bunch of tag-related code in bulk.js. I wonder if anything else can/should be ditched here.